mlflow/server/handlers.py

```diff
@@ -26,21 +27,41 @@
 STATIC_PREFIX_ENV_VAR = "_MLFLOW_STATIC_PREFIX"
 
 
+def string_to_literal(string):
+    if string == 'true':
+        return True
+
+    if string == 'false':
+        return False
+
+    try:
+        return ast.literal_eval(string)
```

Comment:
This seems pretty scary since it may change across Python versions. If we have to do this, it would be better to whitelist a few types, ideally just booleans and strings or something like that.